configs/3dssd/3dssd_kitti-3d-car.py

```diff
@@ -87,8 +87,8 @@
 ]
 
 data = dict(
-    samples_per_gpu=4,
-    workers_per_gpu=4,
+    samples_per_gpu=2,
```

Comment:
This config file seems strange to me. It doesn't have `batch_size x num_GPUs` in its name (such as `2x8`)?

Reply:
I forget to modify it, sorry

Reply:
What about the filename? Should we change it to something like `3dssd_**4x8**_kitti-3d-car.py`?